### DISCUSSION TITLE
How do we get access to new INBOUND endpoints ?

### DISCUSSION THREAD

**Discussion by alexartisans** [amzn/selling-partner-api-models#3836] [General] [answered]
Seems any call returns "X" at the moment. We are greatly affected by 200 SKU limit and new Inbound is something that we have to do ASAP., And now we have to "apply for access" ??? Where do you do that ? We don't have "public application" !

**Comment by MeenaAmz** [comment]
Do you already have a SP-API application? If you don't, the first step is to create a private SP-API application [create a private SP-API application](https://developer-docs.amazon.com/sp-api/docs/tutorial-create-a-private-selling-partner-api-application). Then please follow this [announcement](https://developer-docs.amazon.com/sp-api/changelog/get-early-access-to-new-fulfillment-inbound-api-v2024-03-20) for more details.

**Comment by alexartisans** [comment]
Yes We DO. However "Inbound API v2024-03-20" returns "X" with every call.
Is there a "separate application process" for this one ?

**Comment by MeenaAmz** [comment, marked as answer]
There is no special process. You should be able to access these APIs like other SP-APIs. One thing I would recommend is that you check if you have access to "Amazon Fulfillment" role and the "Product Listing" role in your developer profile. If you do and your still getting 'X', please provide more details on which operation you are using, request details and the response code and message?

**Comment by alexartisans** [comment]
Sorry, we messed up the body and your error processing is not existing... But yes, it works... Off to the races now. Just very upsetting story, really. We spent lots of time integrating v0. And just when we were done - instead of improving v0 - you issued a brand new API... Also still no AWD ? Is this even on the roadmap ?

**Comment by MeenaAmz** [comment]
Thanks for confirming. FBA inbound v1 APIs closely mirrors the send to amazon workflow in seller central unlike v0. We will post when we have updates on AWD.

**Reply by pacton** [reply]
@MeenaAmz - This is simply not true. The Fulfillment Inbound is a completely different animal. Nothing is the same. You probably have discovered that by now. It is buggy as ever. We are waiting 1+ week on Level 2 support case responses. CRAP. They put a deadline date on this when they aren't even close to being beta themselves!
